app/helpers/questions_helper.rb

```diff
@@ -11,5 +11,15 @@ def post_question question
     response = RestClient.post(ENV["qm_api_url"] + 'questions', json, :content_type => :json , :accept => :json)
     logger.debug "THIS IS THE #{response}"
   end
+
+  def next_question question_list, current_question_id
+    # find index of current_question_id in question_list
+    # return current_question_list[index+1]
+  end
+
+  def prev_question
+    question. where("id < ?", id).last
+  end 
+
```

Comment:
This bundle of work doesn't include the `next` and `previous` functionality, so can you remove these changes before we commit it to the main project?